### PR TITLE
Bridge Monitor

### PR DESCRIPTION
**Pull request by janet-harmony**
Monitor the total balance between exchanged tokens
Notify PagerDuty if balance varies too much from golden value

Deployed on HarmonyEngRoom machine
Check status of daemon: systemctl status bridge-watchd

### CODE REVIEW

File: monitoring/bridge-monitor/t.txt [hunk 1]
```diff
@@ -0,0 +1,29 @@
+[
```
Comment (fxfactorial) [inline monitoring/bridge-monitor/t.txt]: Remove this file from the PR -

File: monitoring/bridge-monitor/golang-notes.md [hunk 2]
```diff
@@ -0,0 +1,799 @@
+
```
Comment (fxfactorial) [inline monitoring/bridge-monitor/golang-notes.md]: Remove this file from PR

**Comment by fxfactorial** [comment]
Looks great! Great work! In addition to files asked to remove, also remove  monitoring/bridge-monitor/bnbcli

File: monitoring/bridge-monitor/t.txt (hunk 1 above)
Comment (janet-harmony) [reply monitoring/bridge-monitor/t.txt]: Removed

File: monitoring/bridge-monitor/golang-notes.md (hunk 2 above)
Comment (janet-harmony) [reply monitoring/bridge-monitor/golang-notes.md]: Removed

**Comment by fxfactorial** [comment]
Fantastic work!

**Pull request merged**